cgroups/fs/apply_raw.go

```diff
@@ -24,6 +24,11 @@ var (
 	CgroupProcesses = "cgroup.procs"
 )
 
+type Manager struct {
+	Cgroups *cgroups.Cgroup
+	paths   map[string]string
```

Comment:
nit: Can we add a comment for this being a map from what to what? I believe it is from cgroup subsystem to path at which it is mounted.